src/components/cards/MyPoolBalancesCard/MyPoolBalancesCard.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+import { computed, toRef } from 'vue';
+import { FullPool } from '@/services/balancer/subgraph/types';
+import { bnum } from '@/lib/utils';
+import PoolCalculator from '@/services/pool/calculator/calculator.sevice';
+// Composables
+import useTokens from '@/composables/useTokens';
+import useNumbers from '@/composables/useNumbers';
+import useUserSettings from '@/composables/useUserSettings';
+// Components
+import AssetRow from './components/AssetRow.vue';
+
+/**
+ * TYPES
+ */
+type Props = {
+  pool: FullPool;
+};
+/**
+ * PROPS & EMITS
+ */
+const props = defineProps<Props>();
+
+/**
+ * COMPOSABLES
+ */
+const { tokens, balances, balanceFor } = useTokens();
+const { fNum, toFiat } = useNumbers();
+const { currency } = useUserSettings();
+
+/**
+ * SERVICES
+ */
+const poolCalculator = new PoolCalculator(
+  toRef(props, 'pool'),
+  tokens,
+  balances,
+  'exit'
+);
+
+/**
+ * COMPUTED
+ */
+const bptBalance = computed((): string => balanceFor(props.pool.address));
```

Comment:
Is there a difference between this way of typing and `computed<string>`? (I've mostly been using it when it isn't inferred correctly.